contexts/credentials/v2

```diff
@@ -87,6 +87,69 @@
           "@container": "@graph"
         }
       }
+    },
+
+    "DataIntegrityProof": {
+      "@id": "https://w3id.org/security#DataIntegrityProof",
+      "@context": {
+        "@protected": true,
+        "id": "@id",
+        "type": "@type",
+        "challenge": "https://w3id.org/security#challenge",
+        "created": {
+          "@id": "http://purl.org/dc/terms/created",
+          "@type": "http://www.w3.org/2001/XMLSchema#dateTime"
+        },
+        "domain": "https://w3id.org/security#domain",
+        "expires": {
+          "@id": "https://w3id.org/security#expiration",
+          "@type": "http://www.w3.org/2001/XMLSchema#dateTime"
+        },
+        "nonce": "https://w3id.org/security#nonce",
+        "proofPurpose": {
+          "@id": "https://w3id.org/security#proofPurpose",
+          "@type": "@vocab",
+          "@context": {
+            "@protected": true,
+            "id": "@id",
+            "type": "@type",
+            "assertionMethod": {
+              "@id": "https://w3id.org/security#assertionMethod",
+              "@type": "@id",
+              "@container": "@set"
+            },
+            "authentication": {
+              "@id": "https://w3id.org/security#authenticationMethod",
+              "@type": "@id",
+              "@container": "@set"
+            },
+            "capabilityInvocation": {
+              "@id": "https://w3id.org/security#capabilityInvocationMethod",
+              "@type": "@id",
+              "@container": "@set"
+            },
+            "capabilityDelegation": {
```

Comment:
What does this have to do with credentials ?

Reply:
The term definitions here are identical to the ones used in the data integrity context because they are intentionally `@protected` to simplify processing. Using different ones would be confusing and would raise a redefinition error if anyone combined the contexts. This is better for developers.

Reply:
To be clear, this has to do with using this context for things that are not verifiable credentials correct?

Can we at least provide aninformative use case for how this might be related to verifiable credentials?

Some thing like, `capabilityDelegation` can be used to grant access to verifiable credentials stored on remote services?

Reply:
Explanation and plan to move forward is here: https://github.com/w3c/vc-data-model/pull/943#issuecomment-1287491190